bokehjs/src/coffee/models/tools/gestures/wheel_zoom_tool.coffee

```diff
@@ -43,52 +53,14 @@ class WheelZoomToolView extends GestureTool.View
 
     factor  = @mget('speed') * delta
```

Comment:
As long as we are in this code, every standard `@define` property can just be accessed with `@speed` (when `@` is the model) or `@model.speed` (when `@` is the view) No more `get` or `mget` needed